### DISCUSSION TITLE
電梯簡報是什麼?

### DISCUSSION THREAD

**Discussion by youtinglin** [1101-GS4719-C/Python-Community-GS4719-C#1524] [General]
這學期快結束啦~不知道大家是否已經登出，還是準備登出了呢?
想必期末一定除了考試還會有各種神奇的報告或以各種形式呈現的報告。
而我要介紹的**電梯簡報**我認為是一種很好推銷自己才能或表達自身價值和亮點的一種呈現方式，且對於未來職涯有一定的幫助。

### 

> ### 電梯簡報是什麼?為什麼稱作電梯簡報?
會稱為電梯簡報是因為這種報告的形式是在短時間內將自己想敘述的內容精簡化，且能夠清楚理解和吸引到你的目標客群。
也就是說能夠在短時間內達到你的目的，比如說能夠更順利爭取到offer。

> ### 如何做電梯簡報?

- 第一步：要確定你要呈現的對象是誰?希望他能滿足你什麼事?
- 第二步：如何吸引你的聽眾?主要的問題或內容重點?
- 第三步：說明聽眾對你的重要性?敘述未來目標，或可創造的價值。

**如果內容有專業術語需要補充說明，且描述要直白一點**

希望這篇分享能對大家有所幫助~
[https://www.thebiologist.org/blog/230527-how-to-elevator-pitch-science-speech-example?ss_source=sscampaigns&ss_campaign_id=64862a6cb6062d6119861dfd&ss_email_id=6486602c7771ba258fa4b542&ss_campaign_name=Stay+in+the+Know.+And+More%EF%BD%9C%E5%85%AD%E6%9C%88%E7%AF%87%E3%80%82&ss_campaign_sent_date=2023-06-12T00%3A00%3A55Z](url)
這個連結供大家參考(裡面有更詳細的介紹)

**Comment by Chiuhsianglin** [comment]
做好一個簡報是很重要的，從小到大都一定會有上台報告的時候，小時候可能想說把資訊放上去就好了，內文可能有一大篇文字，沒有重點，導致別人在聽你的報告的時候很常會感到很疲累，所以我認為電梯簡報是一個很好的教材！透過他的步驟做出一個好的簡報。
